bin/rose-make-docs

```diff
@@ -175,7 +175,7 @@ venv-activate () {
 
 venv-install () {
     venv-destroy
-    virtualenv --python=python2.7 "${VENV_PATH}"
+    virtualenv --python=python3.7 "${VENV_PATH}"
```

Comment:
Is there a documentation dependency need for this to be ``3.7`` rather than ``3.6``? With the current PR change set, ``rose check-software`` will specify a minimum of ``3.6``, so we should change either of these for consistency based on requirements. Though looking at ``etc/rose-docs-env.yaml`` where ``3.6`` has been specified I suspect ``.7`` may just be a typo?

Reply:
Fixed this - thanks for bringing it to my attention. 
I am concerned that we ought to be testing against 3.7 anyway - @oliver-sanders  has already found a future-deprecation warning in 3.7 from my updated code.